lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
                     "transition_to": "pending_review",
+                    "attributes": {
```

Comment:
I don't think these are used. Can you remove the `attributes` hashes from this file?